lib/code_corps/map_utils.ex

```diff
@@ -8,7 +8,6 @@ defmodule CodeCorps.MapUtils do
   def keys_to_string(map), do: stringify_keys(map)
 
   # Goes through a list and stringifies keys of any map member
-  defp stringify_keys(nil), do: nil
   defp stringify_keys(%DateTime{} = val), do: val
```

Comment:
Just for anyone visiting this thread in the future.

The problem is that 

```
stringify_keys(val) when is_map(val) do
```

clause needs to go first, in order to match properly against a map at all times. 

Our App used to transform large maps which also include `DateTime` structs. To elixir, a struct is a map, so the code here would try to convert `DateTime` struct keys recursively, which was causing issues, so we needed a `DateTime` clause before the map clause.

However, now our code is no longer using `DateTime` structs within its payloads, so it should be save to remove. We don't need the clause on line 11 anymore.

The clause on line 19 is a duplicate of line 22, so we don't need 19 either.

For discussions sake, I thought about this and I believe we could match against all structs using

```Elixir
defp stringify_keys(%{_struct_: _} = val), do: val
```

on line 11, but I don't think it's necessary for us to do that right now. I believe it's enough to just remove lines 11 and 19 and then merge as is.